src/instructlab/generator/generate_data.py

```diff
@@ -59,6 +59,27 @@
 {% endif -%}
 """
 
+DEFAULT_PROMPT_TEMPLATE_MIXTRAL= """\
+You are asked to come up with a set of 5 diverse questions under {{taxonomy}} - {{task_description}}.
+
+Please follow these guiding principles when generating responses:
+* Use proper grammar and punctuation.
+* Always generate safe and respectful content. Do not generate content that is harmful, abusive, or offensive.
+* Always generate content that is factually accurate and relevant to the prompt.
+* The questions should be clear and human-like.
+* The questions should be diverse and cover a wide range of topics.
+* The questions should not be template-based or generic, it should be very diverse.
+* Simply return the questions, do not return any answers or explanations.
+* Strictly adhere to the prompt and generate responses in the same style and format as the example.
+Use this format to generate the questions: 
+### Question 1: 
+
+Now generate 5 questions following the example below, remember to follow the principles mentioned above and use the same format as the examples. Remember to use the same style and format as the example below. 
+
+To better assist you with this task, here is an example:
+### Question 1:
+"""
+
```

Comment:
```suggestion
DEFAULT_PROMPT_TEMPLATE_MIXTRAL = """\
<s> [INST]You are a very knowledgeable AI Assistant that will faithfully assist the user with their task. You are asked to come up with a set of 5 diverse task instructions under {{taxonomy}}{{" for the task \\"%s\\""|format(task_description)  if task_description}}. These task instructions will be given to a GPT model and we will evaluate the GPT model for completing the instructions.

Here are the requirements:
1. Try not to repeat the verb for each instruction to maximize diversity.
2. The language used for the instruction also should be diverse. For example, you should combine questions with imperative instructions.
{% if not document -%}
3. The type of instructions should not have topic diversity. The list should follow the same topic and category.
{% else -%}
3. The type of instructions should be similar to provided examples. The generated instruction and the output should be grounded in the provided document.
{% endif -%}
4. A GPT language model should be able to complete the instruction. For example, do not ask the assistant to create any visual or audio output. For another example, do not ask the assistant to wake you up at 5pm or set a reminder because it cannot perform any action.
5. The instructions should be in English.
6. The instructions should be 1 to 2 sentences long. Either an imperative sentence or a question is permitted.
{% if not document -%}
7. You should generate an appropriate input to the instruction. The input field should contain a specific example provided for the instruction. It should involve realistic data and should not contain simple placeholders. The input should provide substantial content to make the instruction challenging but should ideally not exceed 100 words.
8. Not all instructions require input. For example, when an instruction asks about some general information, "what is the highest peak in the world", it is not necessary to provide a specific context. In this case, we simply put "<noinput>" in the input field.
9. The output should be an appropriate response to the instruction and the input. Make sure the output is less than 100 words.
{% else -%}
7. The output should be an appropriate response to the input and the instruction. Long outputs are preferable.
{% endif %}

{% if not document -%}
List of 5 tasks:
{% else -%}
Based on below document provide a list of 5 tasks:

Document:
{{document}}

Here are some examples to help you understand the type of questions that are asked for this document:
{% endif -%}[/INST]
"""

```